<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsMeterModeSettings</class>
 <widget class="QDialog" name="clsMeterModeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>315</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>测试设定</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	
	background-color: rgb(148, 148, 248);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>触发方式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbSoftWare">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>软件触发</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAdu200">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ADU200 触发,忙碌K2,通过K1, 失败K0</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rbAutoDetect">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>自动探测样品触发</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>前延时(ms):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="txtPreDelay">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>后延时(ms):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="txtLastDelay">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>保存结果类型</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="rbAll">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>全部</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbPass">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>通过</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbFail">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>失败</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>序号</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>输入序号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="txtStartNumber">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>显示设定</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="rbSingleRes">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>显示单次
测试结果</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>测试设定</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QCheckBox" name="chkFailPass">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>单步不通过，不进行后续测试</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkFailRetest">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>测试失败再进行一次测试</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>列表分割符：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSp">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>,</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>;</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
